src/styles/layouts/app-layout.ts

```diff
@@ -13,19 +13,39 @@ export const useEditorLayoutStyles = makeStyles({
   },
 
   drawer: {
+    top: 64,
     width: drawerWidth,
-    height: "100%"
+    height: "100%",
   },
 
   drawerPaper: {
     width: drawerWidth,
-    backgroundColor: theme.palette.background.default
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: "0 10px 16px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19)"
```

Comment:
Use template literal in here, it makes the multiple shadows easier to read apart since it allows multiple lines.
```suggestion
    boxShadow: `
      0 10px 16px 0 rgba(0,0,0,0.2),
      0 6px 20px 0 rgba(0,0,0,0.19)
    `
```